spec/sidekiq/evss/disability_compensation_form/form526_document_upload_failure_email_spec.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe EVSS::DisabilityCompensationForm::Form526DocumentUploadFailureEmail, type: :job do
+  let!(:form526_submission) do
+    create(
+      :form526_submission,
+      :with_uploads
+    )
+  end
+
+  let(:upload_data) { [form526_submission.form[Form526Submission::FORM_526_UPLOADS].first] }
+  let(:file) { Rack::Test::UploadedFile.new('spec/fixtures/files/sm_file1.jpg', 'image/jpg') }
+  let!(:form_attachment) do
+    sea = SupportingEvidenceAttachment.new(
+      guid: upload_data.first['confirmationCode']
+    )
+
+    sea.set_file_data!(file)
+    sea.save!
+    sea
+  end
+
+  let(:notification_client) { double('Notifications::Client') }
+
+  before do
+    Sidekiq::Job.clear_all
+    allow(Notifications::Client).to receive(:new).and_return(notification_client)
+  end
+
+  describe '#perform' do
+    let(:formatted_submit_date) do
+      form526_submission.created_at.strftime('%B %-d, %Y %-l:%M %P %Z').sub(/([ap])m/, '\1.m.')
+    end
+
+    it 'dispatches a failure notification email with an obscured filename' do
+      obscured_filename = 'sm_***e1.jpg'
```

Comment:
The `obscured_filename` could be moved up and reutilized in each of the 2 tests in this describe block.

Reply:
Good catch fixed in https://github.com/department-of-veterans-affairs/vets-api/pull/16359/commits/25cabc22575f6b0d8a64daa423af5c99c7cb3f19